src/imgui/ImGuiMachine.hh

```diff
@@ -23,6 +25,8 @@ public:
 public:
 	using ImGuiPart::ImGuiPart;
```

Comment:
Detail: this `using` statement means: make all `ImGuiPart` constructors visible in this class (that happens automatically for regular methods, but not for constructors).

But on the line below you override that constructor again (the constructor that takes an `ImGuiManager` parameter). So this `using` statement is no longer useful, remove it.

Reply:
Thanks, I didn't realize at all that this `using` statement had this meaning in this context. So, learned something again...!

I removed it from the commit.